### ISSUE TITLE
User is unable to tab past password field to select other login options like Trusted Networks

### ISSUE THREAD

**Issue by dshokouhi** [home-assistant/frontend#4689]
## Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.

## The problem

When a user navigates to the HA login and they have more than 1 authentication method they are unable to press the tab key past the password field.  Pressing tab switches between the password text field and the icon to show the password.  It does not allow the user to go to the next enabled button.

## Expected behavior

User should be able to navigate to every enabled button using only a keyboard.

This is a big impact for keyboard only users.

## Steps to reproduce

1. Setup HA with more than 1 authentication method (i.e. Trusted Networks and user accounts)
2. Navigate to HA in any browser
3. Press the tab key to select Trusted Networks

## Environment

- Home Assistant release with the issue: 0.105.0.dev
- Last working Home Assistant release (if known): n/a
- UI Type (States or Lovelace): n/a
- Frontend version: 20200130.0
- Browser and browser version:  Chrome 79
- Operating system: Windows 10

## Problem-relevant configuration

```yaml
homeassistant:
  auth_providers:
    - type: homeassistant
    - type: trusted_networks
      trusted_networks:
        - 127.0.0.1
```

## Javascript errors shown in your browser console/inspector

```txt

```

## Additional information
![image](https://user-images.githubusercontent.com/1634145/73497904-b4fb7d00-4370-11ea-8c61-8d0e79af372f.png)

**Comment by dshokouhi** [comment]
WCAG criteria for this: https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-order.html

**Comment by JeffersonBledsoe** [comment]
I'm interested in moving towards working on accessibility in the future. Any chance I'd be able to take a look at this? May need some help along the way, I've yet to work with web-components
